### ISSUE TITLE
Introduced Bug: Page header text sizing

### ISSUE THREAD

**Issue by Macca2805** [dddwa/dddperth-website#78]
I've just noticed I've introduced a bug where the page header sizing is smaller than it was before.

Sizes should be (before change);

```
XS - 25px
SM - 30px
MD - 35px
LG - 40px
XL - 45px
```

I'll address this in the morning.

**Comment by robdmoore** [comment]
Cool. I’ll hold off deploying to prod then.

Test site is at dddperth-website-test.azurewebsites.net<http://dddperth-website-test.azurewebsites.net> FYI


--



On 14 May 2018, at 10:15 pm, Macklin Hartley <notifications@github.com<mailto:notifications@github.com>> wrote:


I've just noticed I've introduced a bug where the page header sizing is smaller than it was before.

Sizes should be (before change);

XS - 25px
SM - 30px
MD - 35px
LG - 40px
XL - 45px


I'll address this in a morning.
